go.mod

```diff
@@ -165,7 +164,7 @@ require (
 	k8s.io/utils v0.0.0-20220210201930-3a6ce19ff2f9 // indirect
 	sigs.k8s.io/json v0.0.0-20211208200746-9f7c6b3444d2 // indirect
 	sigs.k8s.io/structured-merge-diff/v4 v4.2.1 // indirect
-	sigs.k8s.io/yaml v1.3.0 // indirect
+	sigs.k8s.io/yaml v1.2.0 // indirect
```

Comment:
Isn't it a downgrade?

Reply:
🤷 wasn't me. This was introduced by `go tidy`.